catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/services/database/DatabaseServiceResource.java

```diff
@@ -355,4 +356,18 @@ private DatabaseService getService(CreateDatabaseService create, SecurityContext
         .withUpdatedBy(securityContext.getUserPrincipal().getName())
         .withUpdatedAt(System.currentTimeMillis());
   }
+
+  private void validateDatabaseConnection(
```

Comment:
Maybe stupid question, why are we defining this function both in DatabaseServiceResource and DatabaseServiceRepository?

Reply:
left over code. deleting.